home-docs/docker-readme.md

```diff
@@ -42,5 +37,4 @@ These variables allow you to set user and password of your database
 
 `-Xss<size>` - Set thread stack size
 
```

Comment:
Doesn't that set the initial stack size?

Reply:
No, you can't set the initial stack size. This function allowed only for heap size. Docs: https://docs.oracle.com/cd/E13150_01/jrockit_jvm/jrockit/jrdocs/refman/optionX.html#wp1024112